pylock.toml

```diff
@@ -0,0 +1,426 @@
+# This file is @generated by PDM.
+# It is not intended for manual editing.
+lock-version = "1.0"
+requires-python = ">=3.9"
+environments = [
+    "python_version >= \"3.9\"",
+]
+extras = []
+dependency-groups = ["check", "default", "test"]
+default-groups = ["default"]
+created-by = "pdm"
+
+[[packages]]
+name = "coverage"
+version = "7.9.1"
+requires-python = ">=3.9"
+sdist = {name = "coverage-7.9.1.tar.gz", url = "https://files.pythonhosted.org/packages/e7/e0/98670a80884f64578f0c22cd70c5e81a6e07b08167721c7487b4d70a7ca0/coverage-7.9.1.tar.gz", hashes = {sha256 = "6cf43c78c4282708a28e466316935ec7489a9c487518a77fa68f716c67909cec"}}
+wheels = [
+    {name = "coverage-7.9.1-cp313-cp313-macosx_10_13_x86_64.whl",url = "https://files.pythonhosted.org/packages/d0/a7/a027970c991ca90f24e968999f7d509332daf6b8c3533d68633930aaebac/coverage-7.9.1-cp313-cp313-macosx_10_13_x86_64.whl",hashes = {sha256 = "31324f18d5969feef7344a932c32428a2d1a3e50b15a6404e97cba1cc9b2c631"}},
```

Comment:
Unsure why there's a difference in post-comma white space between `sdist` and `wheels` here, but ignoring that for now.